packages/envision/src/js/range-slider.js

```diff
@@ -387,4 +394,28 @@ if (typeof document !== 'undefined') {
    };
 }
 
-export default RangeSlider;
+export default async (elements, settings, args) => {
+   const nodes = getNodes(elements);
+
+   if (nodes.length > 0) {
+      const slider = nodes[0];
+      let rangeSlider;
+      if (rangeSliderMap.has(slider.id)) {
+         rangeSlider = rangeSliderMap.get(slider.id);
+      } else {
+         rangeSlider = new RangeSlider(slider, settings);
+         rangeSliderMap.set(slider.id, rangeSlider);
+      }
+
+      if (typeof settings === 'string') {
```

Comment:
Det här alternativet att skicka in en sträng för att anropa en specifik metod istället för att anropa metoden på det returnerade objektet?

Reply:
Vi har ju det i det gamla, men vi borde kanske ändra när vi gör om?

var rangeSlider = envision.rangeSlider('#example');
rangeSlider.then(function (sliders) {
   sliders[0].setValues([50, 90]);
});

Reply:
Samma för show/hide etc på collapse/accordion också